webxrlayers-1.bs

```diff
@@ -1664,7 +1664,8 @@ This module replaces the steps given by "[=update the pending layers state=]" fr
 <dfn dfn-for="layers">update the pending layers state</dfn> with {{XRSession}} |session| and {{XRRenderStateInit}} |newState|, it must run the following steps:
   1. If |newState|'s {{XRRenderStateInit/baseLayer}} is set:
     1. If the |session| was created with "[=feature descriptor/layers=]" enabled, throw an {{NotSupportedError}} and abort these steps.
-    1. If |newState|'s {{XRRenderStateInit/layers}} is not empty, throw a {{TypeError}} and abort these steps.
+    1. If |newState|'s {{XRRenderStateInit/layers}} is not <code>null</code>, throw a {{NotSupportedError}}.
+    1. Abort these steps.
```

Comment:
Which "these steps" get aborted? I'm not sure what it means to abort an algorithm without throwing an exception.

Reply:
Also, in other steps we've used the language "`x` is set" rather than "`x` is not `null`".

Reply:
> Which "these steps" get aborted? I'm not sure what it means to abort an algorithm without throwing an exception.

That means that you jump back to the algorithm that call this one. That one is defined in the WebXR spec.

Reply:
> Also, in other steps we've used the language "`x` is set" rather than "`x` is not `null`".

I copied "is not null" from the WebXR spec.